assets/src/sass/_essentials.scss

```diff
@@ -3,6 +3,10 @@
 // Foundation essentials.
 @import "~foundation-sites/scss/global";
 @import "~foundation-sites/scss/foundation";
-
 @import "0-settings/settings";
 @import "1-tools/tools";
+
+// Import Zurb Foundation Grid
+@include foundation-flex-grid;
+@include foundation-flex-classes;
+@include foundation-visibility-classes;
```

Comment:
This will load all grid related classes (like .row, column, large, medium, and small, etc.) in all files where we import essentials. This will cause duplicate grid styles on inner pages. We are already loading grid classes in `main.scss` which loads on all pages.

Reply:
aha... missed that. Then I think we should move the `@imports` from foundation to `main`, importing it in `essetinal` won't be required as we don't need that in other files, unless we plan on using other parts of the `foundation`. 🤔